enhancements/olm/platform-operators.md

```diff
@@ -450,11 +452,22 @@ type PlatformOperatorSpec struct {
     PackageName string `json:"packageName"`
 }
 
+// ActiveBundleDeployment references a BundleDeployment resource.
+type ActiveBundleDeployment struct {
+    // Name is the metadata.name of the referenced BundleDeployment object.
+    // +kubebuilder:validation:Required
+    // +required
+    Name string `json:"name"`
+}
+
 // PlatformOperatorStatus defines the observed state of PlatformOperator.
 type PlatformOperatorStatus struct {
     // Conditions represent the latest available observations of a platform operator's current state.
     // +optional
     Conditions []metav1.Condition `json:"conditions,omitempty"`
+    // ActiveBundleDeployment is the reference to the BundleDeployment resource that's
+    // being managed by this PO resource.
+    ActiveBundleDeployment *ActiveBundleDeployment `json:"activeBundleDeployment,omitempty"`
```

Comment:
The field shouldn't be a pointer, and should have an optional tag

Reply:
Updated in https://github.com/openshift/enhancements/pull/1170/commits/9ad65a0426f3132634b59f3694f5e28976c09a28.